src/libs/core/src/compiler.cpp

```diff
@@ -405,7 +459,11 @@ void COMPILER::LoadPreprocess()
         else
             bRuntimeLog = true;
 
-        use_script_cache_ = engine_ini->GetInt("script", "use_cache", false);
```

Comment:
We could still use the `use_cache` value as a fallback for when `cache_mode` is not defined, but since scripting caching is a fairly new feature (I think?) I don't think backwards-compatibility is important.